scripts/ci/vagrant.sh

```diff
@@ -70,6 +70,10 @@ fedora-rawhide() {
 	#
 	ssh default 'sudo dnf remove -y crun || true'
 	ssh default sudo dnf install -y podman runc
+	# Some tests in the container need selinux to be disabled.
+	# In the container it is not possible to change the state of selinux.
+	# Let's just disable it for this test run completely.
+	ssh default 'setenforce Permissive'
```

Comment:
Should it be `sudo setenforce Permissive`?
```
+ ssh default 'setenforce Permissive'
setenforce:  security_setenforce() failed:  Permission denied
make: *** [Makefile:73: vagrant-fedora-rawhide] Error 2
```

Reply:
Yes. Thanks. Changed.